lib/hawkular/metrics/metric_api.rb

```diff
@@ -69,6 +69,18 @@ def query_stats(gauge_ids: [], counter_ids: [], avail_ids: [], rates: false, sta
       http_post(path, data)
     end
 
+    # Fetch all tags for metrics definitions
+    # @return [Hash{String=>String}]
+    def tags
```

Comment:
Would it make sense to allow limiting by key - e.g. 'only return tags that start with app:' ?